java/src/jmri/TurnoutManager.java

```diff
@@ -264,4 +265,29 @@
 
     public String getDefaultClosedSpeed();
 
+    /**
+     * Get the Interval (in ms) to wait between commands.
+     * Configured in AdapterConfig, stored in memo.
+     *
+     * @param systemName turnout systemName, used to identify the manager's connection in ProxyTurnoutManager
+     * @return the (Turnout) Output Interval in Milliseconds
+     */
+    public int getOutputInterval(@Nonnull String systemName);
+
+    /**
+     * Set the Interval (in ms) to wait between commands.
+     *
+     * @param systemName turnout systemName, used to identify the manager's connection in ProxyTurnoutManager
+     * @param newInterval the new Output Interval in Milliseconds
+     */
+    public void setOutputInterval(@Nonnull String systemName, int newInterval);
```

Comment:
It's confusing that this is setting a time that's by-manager using a by-bean syntax (the systemName parameter). Since it's a rare configuration thing (or at least should be), why not directly set it on the manager?  Proxymanagers could just warn and ignore.

Reply:
To find the correct manager via the Proxymanger, I need only the prefix part of the ``systemName``. This might show that I have misunderstood the Proxymanager's function.

Reply:
But why is this going through the ProxyManager at all?  As I understand it, the default delay is set in the configuration for a specific connection.  In that case, the particular type & instance of TurnoutManager is known already.